File: Blazor/Blazor/Complex.cs
using System;

namespace Blazor
{
    public class Complex
    {
        private double X;
        private double Y;

        public Complex(double x, double y)
        {
            X = x;
            Y = y;
        }
        public double getX()
        {
            return X;
        }
        public double getY()
        {
            return Y;
        }
        public double modul2()
        {
            return Math.Sqrt(X * X + Y * Y);
        }
        public Complex Add(Complex z)
        {
            Complex wynik = new Complex(X + z.X, Y + z.Y);
            return wynik;
        }
        public Complex kwadrat()
        {
            return new Complex(Math.Pow(X, 2) - (Math.Pow(Y, 2)), 2 * X * Y);
        }
    }
}

File: docs/scripts/complex.js
function Complex(xx, yy) {
    this.X = xx;
    this.Y = yy;
    this.getX = () => {
        return this.X
    };
    this.getY = () => {
        return this.Y
    };
    this.modul2 = () => {
        return Math.sqrt((this.X * this.X) + (this.Y * this.Y));
    };
    this.add = (z) => {
        return new Complex(this.X + z.X, this.Y + z.Y)
    };
    this.kwadrat = () => {
        return new Complex(Math.pow(this.X, 2) - (Math.pow(this.Y, 2)), 2 * this.X * this.Y);
    };
}